pkg/api/types.go

```diff
@@ -204,6 +204,8 @@ type VolumeSource struct {
 
 // Similar to VolumeSource but meant for the administrator who creates PVs.
 // Exactly one of its members must be set.
+// All PersistentVolumes are volumes, only some Volumes are persistent.  All pointers
```

Comment:
as per the other PR, yes, this is a valid option, but it doesn't need to be captured in the API at all.